README.md

```diff
@@ -249,16 +264,16 @@ Porffor can run Test262 via some hacks/transforms which remove unsupported featu
 - `test262`: test262 runner and utils
 
 ## Usecases
-Basically none right now (other than giving people headaches). Potential ideas:
+Currently, Porffor is seriously limited in features and functionality, however it has some key benefits:
 - Safety. As Porffor is written in JS, a memory-safe language\*, and compiles JS to Wasm, a fully sandboxed environment\*, it is quite safe. (\* These rely on the underlying implementations being secure. You could also run Wasm, or even Porffor itself, with an interpreter instead of a JIT for bonus security points too.)
 - Compiling JS to native binaries. This is still very early!
+- Inline ASM for when you want to beat the compiler in performance, or just want fine grained functionality.
+- SIMD operations
```

Comment:
Specify this is a future thing?

Reply:
Isn't there already some basic SIMD support? I remember seeing something in `bench`, has that been removed?

Reply:
Oh wait, it looks like it, sorry about that. I'll change that.